ci/logging_tests_controller.yml

```diff
@@ -0,0 +1,184 @@
+    common_pod_status_str: "Running"
+    common_pod_nspace: openshift-operators-redhat
+    common_pod_list:
+      - loki-operator-controller-manager
+
+  tasks:
+    - name: "Verify Pods running"
+      ansible.builtin.import_role:
+        name: common
+        # vars can be passed here, so we can use the same play for this and the next play.
+      # vars:
+      #   - <vars from above>
+
+- name: "Verify logging pods are running in openshift-logging"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12676"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openshift-logging
+    common_pod_list:
+      - cluster-logging-operator
+      - collector
+      - logging-loki-compactor
+      - logging-loki-distributor
+      - logging-loki-gateway
+      - logging-loki-index-gateway
+      - logging-loki-ingester
+      - logging-loki-querier
+      - logging-loki-query-frontend
+      - logging-view-plugin
+
+  ### see JIRA LOG-5431 if pods not running
+  tasks:
+    - name: "Verify Pods running"
+      ansible.builtin.import_role:
+        name: common
+
+- name: "Verify logging pods are running in minio-dev"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12674"
+    common_pod_status_str: "Running"
+    common_pod_nspace: minio-dev
+    common_pod_list:
+      - minio
+
+  tasks:
+    - name: "Run pod running tests"
+      ansible.builtin.import_role:
+        name: common
+
+
+- name: "Verify logging pods have complete status in openstack"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12679"
+    common_pod_nspace: openstack
+    common_pod_status_str: "Completed"
+    common_pod_list:
+      - bootstrap-edpm-deployment-logging
+      - configure-network-edpm-deployment-logging
+      - configure-os-edpm-deployment-logging
+      - download-cache-edpm-deployment-logging
+      - install-certs-edpm-deployment-logging
+      - install-os-edpm-deployment-logging
+      - libvirt-edpm-deployment-logging
+      - logging-edpm-deployment-logging-openstack-edpm
+      - neutron-metadata-edpm-deployment-logging
+      - nova-custom-edpm-deployment
+      - ovn-edpm-deployment-logging
+      - reboot-os-edpm-deployment-logging
+      - repo-setup-edpm-deployment-logging
+      - run-os-edpm-deployment-logging
+      - ssh-known-hosts-edpm-deployment-logging
+      - telemetry-edpm-deployment-logging
+      - validate-network-edpm-deployment-logging
+  tasks:
+    - name: "Run pods completed tests"
+      ansible.builtin.import_role:
+        name: common
+
```

Comment:
You can add another play here to run the container tests OR add the container vars to an existing play (note the play name in line 46) to make the container tests run in the same play.

The reason that the pod vars are all split into different plays is that the pod lists are split by nspace and status string.

I will look into configuring the pod tests with a list like for the endpoint tests [1].

[1] https://github.com/infrawatch/feature-verification-tests/pull/113/files#diff-64b93d86b5903b16bb6cdee44d150891ead9ec2ac9383c8f001cb611fa7958e7R61-R77